File: Linguagens_Programação/C/3.c
//3. Dado um número inteiro positivo n, imprimir os n primeiros naturais ímpares.
#include <stdio.h>
int main()
{
    int N;
    int cont = 1;
    printf("Digite um inteiro N: ");
    scanf("%i", &N);
   
    if(N <= 0){
        printf("Digite um valor válido, positivo!");
        return 0;
    }
    while (cont<=N){
        printf("%i ",cont);
        cont += 2;
    }

    return 0;
}

File: Linguagens_Programação/Go/3.go
/*3. Dado um número inteiro positivo n, imprimir os n primeiros naturais ímpares.*/
package main
import "fmt"

func main() {
    var n int
    var cont int = 1
    fmt.Println("Digite um inteiro N: ")
    fmt.Scanln(&n)
    
    for i := 1; i <= n; i ++ {
        fmt.Printf("%d ", cont)
        cont += 2
    }
}